src/seedu/addressbook/commands/Command.java

```diff
@@ -54,6 +54,13 @@ public void setData(AddressBook addressBook, List<? extends ReadOnlyPerson> rele
         this.relevantPersons = relevantPersons;
     }
 
+    /**
+     *  @return true if command types mutate the data.
```

Comment:
Good job for writing the header comment. BTW, for a method with only return statement, you can instead write it as `Returns true if ...` and remove `@return`.